composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "paysafegroup/paysafe_sdk_php",
+    "description": "Paysafe PHP SDK",
+    "keywords": ["paysafe","php", "netbanx", "optimalpayments", "sdk"],
+    "license": "MIT",
+    "require": {
+        "php": ">=5.3.0"
+    },
+    "autoload": {
```

Comment:
Does purchase need to start w/ a capital p here? `source/paysafe/DirectDebit/purchase.php`